lib/src/native_updater.dart

```diff
@@ -108,4 +86,36 @@ class NativeUpdater {
       },
     );
   }
+
+  void _showMaterialAlertDialog(BuildContext context) async {
+    developer.log('Playstore URL: ${_playStoreUrl ?? ''}');
+
+    /// In App Update Related
+    try {
+      AppUpdateInfo _updateInfo = await InAppUpdate.checkForUpdate();
```

Comment:
@JeffryHermanto I thought we agreed that we will use the direct implementation in the Android side instead of using this `in_app_update` library. May I know what's going on?